clusterman/draining/queue.py

```diff
@@ -109,9 +111,10 @@ def submit_instance_for_draining(
             ),
         )
 
-    def submit_host_for_draining(self, host: Host) -> None:
+    def submit_host_for_draining(self, host: Host, delay: int) -> None:
```

Comment:
I'd set a default None/0 value, so that you can avoid updating upstream calls (e.g. line 390)